controller/controller.go

```diff
@@ -1026,6 +1063,14 @@ func (ctrl *ProvisionController) provisionClaimOperation(claim *v1.PersistentVol
 		}
 		err = fmt.Errorf("failed to provision volume with StorageClass %q: %v", claimClass, err)
 		ctrl.eventRecorder.Event(claim, v1.EventTypeWarning, "ProvisioningFailed", err.Error())
+		if _, ok := claim.Annotations[annSelectedNode]; ok {
```

Comment:
Do we retry on timeout errors?  I remember CSI has a very short CreateVolume rpc timeout, so if provisioning takes more than 10 seconds or so, then it's possible that the rpc will timeout and get retried.

Reply:
sorry, haven't looked at this for a long time, will investigate this.